hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def get_middle(str)
+  hf = str.length / 2
```

Comment:
let's try to name this variable more accurately? `hf` gives not many information 😄